src/main/resources/add-ons-to-index.json

```diff
@@ -317,6 +329,9 @@
       "status": "DEPRECATED",
       "type": "OMOD",
       "name": "Backports 1.4.x",
+      "tags": [
+        "Backports"
```

Comment:
Taking part of the module name and including it as a tag isn't really helpful, kindly try to add meaningful tags.